src/mococrw/openssl_wrap.h

```diff
@@ -209,6 +228,12 @@ using SSL_ECDSA_SIG_SharedPtr = utility::SharedPtrTypeFromUniquePtr<SSL_ECDSA_SI
 
 using time_point = std::chrono::system_clock::time_point;
 
+// Note: ENGINE_free() always returns 1. Therefore, SSLDeleter is suitable as it ignores return
```

Comment:
SSLDeleter is not used?
```suggestion
// Note: ENGINE_free() always returns 1. Therefore, SSLRetDeleter is suitable as it ignores return
```

Reply:
Done.